tests/test_init.py

```diff
@@ -24,17 +24,20 @@
 import talisker
 
 
+def assert_config(env, **kwargs):
```

Comment:
Totally optional, but I'd probably s/kwargs/expected to give the param name proper meaning.